routes/api/v1/meals.js

```diff
@@ -4,39 +4,19 @@ var Food = require('../../../models').Food;
 var Meal = require('../../../models').Meal;
 
 /* GET all meals */
-router.get("/", function(req, res, next) {
-  Meal.findAll()
-    .then(meals => {
-      res.setHeader("Content-Type", "application/json");
-      res.status(200).send(JSON.stringify(meals, ['id', 'name', 'foods']));
-      // res.status(200).send(JSON.stringify(meals, ['id', 'name', 'foods'['id', 'name', 'calories']]));
-    })
-    .catch(error => {
-      res.setHeader("Content-Type", "application/json");
-      res.status(500).send({error})
-    });
-});
 
+router.get('/', (request, response) => {
+  return Meal.findAll({
+    include: Food
```

Comment:
The effect of these 12 characters not including spaces cannot be understated. It took us some time, but I'm glad we got this.